app/models/translation.rb

```diff
@@ -76,6 +81,18 @@ def manifest_translated_phrases
     @manifest_translated_phrases ||= download_translated_phrases("name_description.xml")
   end
 
+  def push_published_to_s3
```

Comment:
We should sort-circuit publishing if the translation has already been published

```suggestion
  def push_published_to_s3
    return if is_published
```